include/aws/common/macros.h

```diff
@@ -131,6 +131,24 @@ enum { AWS_CACHE_LINE = 64 };
 #    define AWS_SUPPRESS_TSAN
 #endif
 
+#if defined(__has_feature)
```

Comment:
hmmm, I am thinking since we already have `AWS_SUPPRESS_TSAN` and `AWS_SUPPRESS_ASAN`, probably makes sense to have this as well. but, yeah, we are not using it for now.